### PR TITLE
Added secure key and logging for createCustomer

### PR DESCRIPTION
**Pull request by heymartinadams**

### CODE REVIEW

File: create/createCustomer.js [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+const secureKey = process.env.SECUREKEY
+
+console.log('Init');
 
 module.exports = async (req, res) => {
   const data = await json(req)
+  const { parse } = require('url');
+  const { query } = parse(req.url, true)
+
+  if (secureKey === query) {
```
Comment (marktani) [inline create/createCustomer.js]: can you add indentation here?

File: create/createCustomer.js [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
   const cardDetails = data.createdNode
   const cardDetailsId = cardDetails.id
 
   const user = cardDetails.user
   const userId = user.id
+//   Add logs during development, but remember to remove them for production
```
Comment (marktani) [inline create/createCustomer.js]: these comments are also not indented correctly

File: create/createCustomer.js (hunk 1 above)
Comment (heymartinadams) [inline create/createCustomer.js]: Sure. Didn’t want to do that at first so you could see only the actual code changes.

Comment (heymartinadams) [inline create/createCustomer.js]: How do I do this, @marktani ? Hardly done pull requests before.

**Comment by heymartinadams** [comment]
Figured it out, @marktani

File: create/createCustomer.js [hunk 3]
```diff
@@ -1,50 +1,63 @@
 const { json, send } = require('micro')
 const request = require('request')
 const stripe = require('stripe')(process.env.STRIPE_SECRET)
+const log = process.env.LOG
+require('now-logs')(log);
```
Comment (marktani) [inline create/createCustomer.js]: I believe you also need to update `package.json`, right?

Comment (heymartinadams) [inline create/createCustomer.js]: Haha, yes 😊 Done.

**Pull request merged**